doc/ref/pres.xml

```diff
@@ -88,8 +88,8 @@ numbering of generators in a family of associative words.
 <Section Label="Printing Presentations">
 <Heading>Printing Presentations</Heading>
 
-Whenever you create a presentation <M>P</M>, say, or assign it to a variable,
-&GAP; will respond by printing <M>P</M>.
+Whenever you create a presentation <M>P</M>, or assign it to a
+variable, &GAP; will respond by printing <M>P</M>.
```

Comment:
Optional and unimportant, but: could move "variable, " back to the previous line, to reduce the diff